spec/features/idv/steps/in_person/state_id_step_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+require 'axe-rspec'
+
+RSpec.describe 'doc auth IPP state ID step', js: true do
+  include IdvStepHelper
+  include InPersonHelper
+  include VerifyStepHelper
```

Comment:
Is this needed?

```suggestion
```

Reply:
removed with commit ba04a568b2753687c532b0ff07a8e25a2ab99f49